gravityflow/gravityflowflowchart
# Fix Flowchart icon for Gravity Forms 2.5

## Description
Re: [Issue# 66](https://github.com/gravityflow/backlog/issues/66) 
Fix Flowchart and icon location to appear more consistent with the UI.

## Testing instructions
1. Check flowchart icon and toolbar location with Gravity Forms 2.5 beta.
2. Rollback to Gravity Forms 2.4, and confirm nothing breaks.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
